tests/components/jellyfin/const.py

```diff
@@ -8,10 +8,115 @@
 TEST_USERNAME: Final = "test-username"
 TEST_PASSWORD: Final = "test-password"
 
+MOCK_USER_ID = "123"
+MOCK_DEVICE_ID = "Home Assistant"
+MOCK_AUTH_TOKEN = "a4c12c43edef56925ba65328a0e96325"
+
 MOCK_SUCCESFUL_CONNECTION_STATE: Final = {"State": CONNECTION_STATE["ServerSignIn"]}
 MOCK_SUCCESFUL_LOGIN_RESPONSE: Final = {"AccessToken": "Test"}
 
 MOCK_UNSUCCESFUL_CONNECTION_STATE: Final = {"State": CONNECTION_STATE["Unavailable"]}
 MOCK_UNSUCCESFUL_LOGIN_RESPONSE: Final = {""}
 
 MOCK_USER_SETTINGS: Final = {"Id": "123"}
+
+MOCK_ALBUM_NAME: Final = "MockAlbum"
+MOCK_ALBUM_ID: Final = "MockAlbumId"
+
+MOCK_ARTIST_NAME: Final = "MockArtist"
+MOCK_ARTIST_ID: Final = "MockArtistId"
+
+MOCK_TRACK_NAME: Final = "MockTrack"
+MOCK_TRACK_ID: Final = "MockTrackId"
+
+MOCK_TRACK: Final = {
```

Comment:
This looks like something we can store in a json file in a fixture directory and load with our fixture load helper in a pytest fixture function.